app/models/user.rb

```diff
@@ -1,10 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [REQUIRED] Kindly enable `:confirmable` as stated by the project requirement. The `:confirmable` module adds an extra layer of security by verifying the user's email address. It helps ensure that users provide a valid email address and prevents unauthorized users from accessing the application.

Reply:
@Whoistolu , Thank you so much for the review, I appriciate.
Now, I have attached an image below showing that the table Users

![Screenshot from 2023-06-24 14-42-45](https://github.com/Caren-Koroeny/caren_blog-app/assets/55842414/0c4e52af-e8a0-4f1a-98fc-6a51eb496493)
 
- I would really want to know how to go about it, since i have been trying it.